### PR TITLE
fix: Add missing styles to contributors feature and limit number of contributors to display

### PR DESCRIPTION
**Pull request by DanielPalafox**

### CODE REVIEW

**Comment by DanielPalafox** [comment]
[[QA][my-orcid] New functionality for contributors in works not implemented](https://trello.com/c/t8fa5jir/7704-qamy-orcid-new-functionality-for-contributors-in-works-not-implemented)

**Review by leomendoza123 — APPROVED** [review]

File: src/app/record/components/work/work.component.html [hunk 1]
```diff
@@ -131,15 +130,82 @@ <h3 class="orc-font-body-small" i18n="@@works.contributors">
...
-        }}</ng-container>
-        {{ contributor | contributorsPipe }}
-      </div>
+        <p>
+          + {{ work.contributors.length - maxNumberContributorsWorkDetails }}
+          <span i18n="@@works.more">more</span>
+        </p>
+        <p>----</p>
+        <p>
+          <ng-container i18n="@@works.more">
```
Comment (leomendoza123) [inline src/app/record/components/work/work.component.html:184]: This label is not using the right key

**Review by leomendoza123 — CHANGES_REQUESTED** [review]
Just a small translation missing

File: src/app/record/components/work/work.component.html (hunk 1 above)
Comment (DanielPalafox) [reply src/app/record/components/work/work.component.html]: Done!

**Review by amontenegro — APPROVED** [review]

**Pull request merged**